Workbooks/Workloads-AMA/Alerts/Alerts.workbook

```diff
@@ -0,0 +1,921 @@
+              "size": 0,
+              "noDataMessage": "No alerts found in this time range or no alert rules have been configured for this monitoring profile",
+              "exportFieldName": "Severity",
+              "exportParameterName": "Severity",
+              "queryType": 1,
+              "resourceType": "microsoft.resourcegraph/resources",
+              "crossComponentResources": [
+                "value::selected"
+              ],
+              "gridSettings": {
+                "formatters": [
+                  {
+                    "columnMatch": "Severity",
+                    "formatter": 11
+                  },
+                  {
+                    "columnMatch": "Count",
+                    "formatter": 3,
+                    "formatOptions": {
+                      "min": 0,
+                      "palette": "blue",
+                      "aggregation": "Sum"
+                    },
+                    "numberFormat": {
+                      "unit": 17,
+                      "options": {
+                        "style": "decimal",
+                        "maximumFractionDigits": 2
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "State",
+                    "formatter": 1
+                  }
+                ]
+              }
+            },
+            "conditionalVisibility": {
+              "parameterName": "1",
+              "comparison": "isEqualTo",
+              "value": "2"
+            },
+            "showPin": true,
+            "name": "Alert Summary"
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "{\"version\":\"1.0.0\",\"content\":\"[\\r\\n\\t{\\r\\n\\t\\t\\\"severity\\\": \\\"Sev0\\\"\\r\\n\\t},\\r\\n\\t{\\r\\n\\t\\t\\\"severity\\\": \\\"Sev1\\\",\\r\\n\\t\\t\\\"new\\\": 0,\\r\\n\\t\\t\\\"acknowledged\\\": 0,\\r\\n\\t\\t\\\"closed\\\": 0\\r\\n\\t},\\r\\n\\t{\\r\\n\\t\\t\\\"severity\\\": \\\"Sev2\\\"\\r\\n\\t},\\r\\n\\t{\\r\\n\\t\\t\\\"severity\\\": \\\"Sev3\\\"\\r\\n\\t},\\r\\n\\t{\\r\\n\\t\\t\\\"severity\\\": \\\"Sev4\\\"\\r\\n\\t}\\r\\n]\",\"transformers\":[{\"type\":\"jsonpath\",\"settings\":{\"columns\":[{\"path\":\"$.severity\",\"columnid\":\"Severity\"}]}}]}",
```

Comment:
nit: don't need anything other than severity column

Reply:
yeah, i'm just using the severity column to merge/show empty severity. the others could be removed, however, i'm still trying to get the links to show up on the no data ones